workspaces/ui-v2/src/pages/changelog/ChangelogEndpointRootPage.tsx

```diff
@@ -260,6 +343,7 @@ const useStyles = makeStyles((theme) => ({
   bodyContainer: {
     marginTop: theme.spacing(6),
     width: '100%',
+    height: '100%',
   },
```

Comment:
These kinds of props on containers worry me a bit (most likely a min-height: 100vh is what's actually wanted), but happy if it's not doing anything particularly funky in this case, a bit of code that can survive another day!

Reply:
The reason we need this, is so the divs inherit the parent height such that the `position: sticky` panel can get a height